lib/WP_Auth0_Options.php

```diff
@@ -108,12 +108,13 @@ public function get_web_origins() {
 	 * Get the main site URL for Auth0 processing
 	 *
 	 * @param string|null $protocol - forced URL protocol, use default if empty
+	 * @param bool $implicit - use the implicit flow in the callback
 	 *
 	 * @return string
 	 */
-	public function get_wp_auth0_url( $protocol = null ) {
+	public function get_wp_auth0_url( $protocol = null, $implicit = false ) {
 		$site_url = site_url( 'index.php', $protocol );
-		return add_query_arg( 'auth0', '1', $site_url );
+		return add_query_arg( 'auth0', ( $implicit ? 'implicit' : '1' ), $site_url );
```

Comment:
how are you handling this with a query param?

Reply:
Determines the login flow to use:

https://github.com/auth0/wp-auth0/blob/dev/lib/WP_Auth0_LoginManager.php#L171